### ISSUE TITLE
Will this project will support the Pytorch?

### ISSUE THREAD

**Issue by Noahsark** [alibaba/euler#1]
Excellent work!

May I kindly ask if this project will support the Pytorch? 

Thank you so much.

**Comment by renyi533** [comment]
Thanks for your interest. Theoretically, it should be straightforward to support Pytorch as Euler exposes native C++ interface. We can easily extend Pytorch Operators. The decision will be based on customer's need. We may support Pytorch if we collect enough feature requests. Alternatively, we also welcome external code contributions. :)

Currently, we support Tensorflow and our X-DeepLearning framework. Considering TF's popularity, our open source version just support Tensorflow now.

**Comment by yangsiran** [comment]
Currently, code of euler core is written using C++ 11 and test on CentOS 7 with gcc 4.8.5. Support on CentOS 6 will need to backport the code base older version of glibc and libstdc++.

**Comment by renyi533** [comment]
Yes. We definitely will support TF 2.0 soon once the stable version is officially released.

**Comment by HuangZhanPeng** [comment]
@yangsiran @renyi533  Got it, thank you very much

**Comment by aimetrics** [comment]
> Yes. We definitely will support TF 2.0 soon once the stable version is officially released.

Hi, @renyi533 Is TF2.0 supported Now? Thanks!

**Comment by weberxie** [comment]
Any news about this question? Looking forward to the support of PyTorch.
